src/modm/platform/can/windows_hosted/module.lb

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016-2018, Niklas Hauser
+# Copyright (c) 2017, Fabian Greif
+# Copyright (c) 2018, Christopher Durand
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+from collections import namedtuple
+from enum import Enum
+
+def load_options(module):
+    module.add_option(
```

Comment:
Are these options actually used? Or are we including them for compatibility in some way?

Reply:
these options are used so in the `project.xml` one can ask for `can:1` or `can:2`

Reply:
I was more referring to the one below called "buffer.rx"

Reply:
Yes, this is for compatibility with the other build. Allows someone in the `project.xml` to specify buffer length. Probably not necessary for what we are doing but it means we won't have to change the `.xml` if we ever do start specifying buffer size.